client/settings.jsx

```diff
@@ -0,0 +1,60 @@
+const React = require('react'),
+      Avatar = require('./avatar.jsx'),
+      Select = require('./select.jsx');
+
+const Settings = React.createClass({
+    getInitialState() {
+        return {
+            sex: ['Αγόρι', 'Κορίτσι', '-'],
+            cities: ['Αθήνα', 'Θεσσαλονίκη']
+        };
+    },
+    render() {
+        return (
+            <div className='form-group'>
+                <div id='settings'>
+                    <div id='icon-username'>
+                        <h2>{this.props.username}</h2>
+                        <div className='icon'>
+                            <Avatar username={this.props.username} />
+                            <span>Αλλαγή της εικόνας μου</span>
+                        </div>
+                    </div>
+
+                    <div id='form'>
+                        <p>
+                            <label htmlFor='password'>Κωδικός</label>
+                            <input className='form-control' type='password' id='password' />
+                        </p>
+
+                        <p>
+                            <label htmlFor='email'>E-mail</label>
+                            <input className='form-control' type='text' id='email' />
+                        </p>
+
+                        <p>
+                            <label htmlFor='birth'>Ημερομηνία Γέννησης</label>
+                            <Select classProp='day' idProp='birth' start={1} end={31} />
+                            <Select classProp='month' start={1} end={12} />
+                            <Select classProp='year' start={1920} end={2017} />
```

Comment:
This may be good for general date picking (e.g. events near today), but perhaps not for date of birth, given that one would have to scroll back quite a few years.